tests/flows/dummy_learn/__init__.py

```diff
@@ -0,0 +1 @@
+__version__ = 1.0
```

Comment:
When I execute the tests locally under python3, it imported the module as `dummy_module.dummy_forest`, on travis-ci as `tests.flows.dummy_learn.dummy_forest`. I changed the imports now, so this is no longer needed.